nolearn/lasagne/base.py

```diff
@@ -571,6 +571,15 @@ def initialize_layers(self, layers=None):
                 else:
                     layer_kw['incoming'] = layer
 
+            # Deal with additional string parameters that may
+            # reference other layers; currently only 'mask_input'.
+            layer_ref_params = ['mask_input']
```

Comment:
@BenjaminBossan Care to pull this out into a class variable?  Would be easier to override that way, if another layer and name ever came up.